clang/lib/Basic/Cuda.cpp

```diff
@@ -121,6 +121,7 @@ static const OffloadArchToStringMap arch_names[] = {
     GFX(909),  // gfx909
     GFX(90a),  // gfx90a
     GFX(90c),  // gfx90c
+    {OffloadArch::GFX9_4_GENERIC, "gfx9-4-generic", "compute_amdgcn"},
```

Comment:
Why "9-4" and not "9.4"?

Reply:
I guess this precedent is already set in 10-1, but I think this was a mistake

Reply:
In fact, `gfx10-1-generic` was called `generic10.1-generic`, but that broke things like save temps, so all generic targets were renamed.